src/base/path.cpp

```diff
@@ -73,15 +81,28 @@ Path::Path(const std::string &pathStr)
 
 bool Path::isValid() const
 {
+    // does not support UNC path
+
     if (isEmpty())
         return false;
 
+    // https://stackoverflow.com/a/31976060
 #if defined(Q_OS_WIN)
-    const QRegularExpression regex {u"[:?\"*<>|]"_qs};
+    QStringView view = m_pathStr;
+    if (hasDriveLetter(view))
+        view = view.mid(3);
+
+    // \\37 is using base-8 number system
+    const QRegularExpression regex {u"[\\0-\\37:?\"*<>|]"_qs};
+#if (QT_VERSION >= QT_VERSION_CHECK(6, 1, 0))
```

Comment:
Maybe use `QRegularExpression::match()` to avoid conditional branches?

Reply:
OK, done.